test-cypress/cypress/commands/api/camera.d.ts

```diff
@@ -0,0 +1,16 @@
+// load the global Cypress types
+/// <reference types="cypress" />
+
+declare namespace Cypress {
+    interface Chainable {
+        /**
+         * create a group for the given user (who has already been referenced in the test
+        */
+        apiCreateCamera(cameraname: string, groupname: String, log?: boolean): Chainable<Element>
```

Comment:
Can we be consistent with our lowerCamelCase naming scheme?  `cameraname` -> `cameraName`, `groupname` -> `groupName` etc